### PR TITLE
fir header dialog information into min width

### LINKED ISSUE #568: Log header does not adapt to the minimum resolution
### Describe the bug

The log header window now has a lot of "horizontal" fields. In older version it was calculated to be usable with the minimum size allowed, but now, it simply cuts the right part:

image

I don't know if we want to "adapt" it to fit (maybe some flex to move the fields from the right to the bottom), give it a horizontal scroll, or change the minimum width of this window.

### To Reproduce

Open the log header, and make the log header window as small as possible.

### Expected behavior

Be able to view all the fields of the log header at minimum resolution.

### Betaflight Blackbox Explorer version

3.6.0 v20220213.5962

### Add any other context about the problem that you think might be relevant here

_No response_

### PR DESCRIPTION
**Pull request by ctzsnooze**
fixes #568

@McGiverGim expressed concern that at minimum window width, not all the information in the header dialog was visible.

I rearranged things a bit - margins, rearranging the header, that sort of thing, and it all now fits quite neatly.

The minimum window width is now 930px, a meaningful increase over the previous minimum of 740px.  The minimum width could be as small as 900px and still look OK (see below), but at 930px the header dialog sits symmetrically above the main graphical area and looks good.  It should still fit into most computer screens at 930px.

Here is an image showing how it looks at 930px width.

redraft min window

This is the previous appearance at about the same width.

same size previously

This is the new appearance but with a min-width of 900px (this is not the value in the PR, but I included it here in case you think it might be better like that).  

We don't really need all the white space at the top but I couldn't figure out how to get rid of it.  

moin width 900

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by Asizon** [comment]
Thankyou @ctzsnooze @haslinghuis!!

**Pull request merged**
